app/models/concerns/sdg/relatable.rb

```diff
@@ -26,6 +28,18 @@ def by_sdg_related(sdg_class, code)
 
       joins(sdg_class.table_name.to_sym).merge(sdg_class.where(code: code))
     end
+
+    def by_revision_status(revision_status)
+      case revision_status
+      when "pending", nil
+        revised_ids = SDG::Revision.where(relatable_type: self.name).pluck(:relatable_id)
+        where.not(id: revised_ids)
```

Comment:
```suggestion
        left_joins(:sdg_revision).merge(SDG::Revision.where(id: nil))
```

Or something similar?